search/includes/classes/commands/class-corecommand.php

```diff
@@ -383,11 +388,8 @@ public function get_index_settings( $args, $assoc_args ) {
 	 * Throw error when delete-index command is attempted to be used.
 	 *
 	 * @subcommand delete-index
-	 *
-	 * @param array $args Positional CLI args.
-	 * @param array $assoc_args Associative CLI args.
```

Comment:
Now that the  method args have been kept, then these (multiple) DocBlocks no longer need the `@param` tags removing.